EnglishFilter/sections/whitelist.txt

```diff
@@ -52,6 +52,9 @@ youtube.com#@##header.ytd-c4-tabbed-header-renderer:style(position: absolute !im
 ! https://github.com/AdguardTeam/AdguardFilters/issues/70664
 !+ PLATFORM(ext_ff)
 linkdrop.net$@$script[tag-content="absda"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/76306
+@@||powr.io/powr.js?platform=shopify$domain=ipitaka.com
+@@||powr.io/form-builder/u$domain=ipitaka.com
```

Comment:
It's caused by the rule `||powr.io^$third-party` from the Annoyances. Why do you add exceptions to Base filter? It should be Annoyances - `push_notifications.txt` and we have such section `! Fixing the issues` at the end of the file.

The rules can also be rewritten something like

```
@@||powr.io/powr.js$domain=ipitaka.com
@@||powr.io/form-builder/$domain=ipitaka.com
```